recipes/r-forestfloor/build.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+if [[ $target_platform =~ linux.* ]] || [[ $target_platform == win-32 ]] || [[ $target_platform == win-64 ]] || [[ $target_platform == osx-64 ]]; then
+  export DISABLE_AUTOBREW=1
+  X11_CONFIGURE_ARGS="--x-includes=${PREFIX}/include --x-libraries=${BUILD_PREFIX}/x86_64-conda_cos6-linux-gnu/sysroot/usr/lib64"
```

Comment:
Is this and the LD_LIBRARY_PATH in the meta.yaml tests still necessary? I applied the yum_requirements.txt fix to https://github.com/conda-forge/r-oceanview-feedstock/pull/2 (failed with the `libGL not found` error as well) and it built successfully without additional changes.

Reply:
@cbrueffer done!

Reply:
@cbrueffer done